src/components/SpecialResolution/ResolutionEditor.vue

```diff
@@ -244,11 +244,7 @@ export default class ResolutionEditor extends Vue {
 
   /** The maximum resolution date that can be entered (today). */
   get resolutionDateMax (): string {
-    if (this.getSpecialResolution.signingDate) {
-      return DateUtilities.dateToYyyyMmDd(DateUtilities.yyyyMmDdToDate(this.getSpecialResolution.signingDate))
-    } else {
-      return this.getCurrentDate
-    }
+    return this.getCurrentDate
```

Comment:
👍 

Even though this computed function is trivial, it's alongside resolutionDateMin(), so there's a nice symmetry / consistency when I look for what the code does.